docs/topics/logging.txt

```diff
@@ -584,6 +584,12 @@ Logs the SQL queries that are executed during schema changes to the database by
 the :doc:`migrations framework </topics/migrations>`. Note that it won't log the
 queries executed by :class:`~django.db.migrations.operations.RunPython`.
 
+.. versionadded:: 1.10
```

Comment:
Putting these markers in the middle of several paragraphs can be ambiguous. I prefer if you put it at the bottom of the section with a note like "The extra context was added."